libraries/typescript/github.com/redis/redis-om-node.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "Redis OM for Python",
+    "name": "Redis OM for Node.js",
     "description": "Object mapping, and more, for Redis and Node.js.",
```

may or may not, just a thought
```suggestion
    "description": "Object mapping, and more, for Redis and Node.js. Written in TypeScript.",
```

---

Hey, I am not sure if it matters that it's written in TypeScript. I don't think it is needed to be specified.

Please overwrite if you think it is needed. :)